modin/pandas/test/test_io.py

```diff
@@ -1858,13 +1848,13 @@ def test_to_feather(self):
 
 
 class TestClipboard:
-    @pytest.mark.skip(reason="No clipboard on Travis")
+    @pytest.mark.skip(reason="No clipboard in CI")
```

Comment:
Should we make this an xfail as well?

Reply:
I don't think we need to. Clipboard requires extra dependencies we don't have.